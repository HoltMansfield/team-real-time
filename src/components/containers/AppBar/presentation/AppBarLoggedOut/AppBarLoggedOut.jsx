import React from 'react'
import { Link } from 'react-router-dom'
import AppBar from 'material-ui/AppBar'
import IconMenu from 'material-ui/IconMenu'
import MenuItem from 'material-ui/MenuItem'
import IconButton from 'material-ui/IconButton'
import NavigationMenuIcon from 'material-ui/svg-icons/navigation/menu'


export default function AppBarLoggedOut(props) {
  return (
    <AppBar
      title="Team Real Time"
      iconElementLeft={
        <IconMenu
          iconButtonElement={<IconButton><NavigationMenuIcon /></IconButton>}
          anchorOrigin={{horizontal: 'left', vertical: 'top'}}
          targetOrigin={{horizontal: 'left', vertical: 'top'}}
        >
          <MenuItem primaryText="About us" />
          <MenuItem primaryText="Create Account" />
          <MenuItem primaryText="Login" />
        </IconMenu>
      }
    />
  )
}
