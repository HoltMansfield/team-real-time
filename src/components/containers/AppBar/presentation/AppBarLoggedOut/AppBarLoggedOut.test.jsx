import React from 'react'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppBarLoggedOut from './AppBarLoggedOut'


it('renders correctly', () => {
  const wrapper = shallow(<AppBarLoggedOut />)

  expect(toJson(wrapper)).toMatchSnapshot()
})
