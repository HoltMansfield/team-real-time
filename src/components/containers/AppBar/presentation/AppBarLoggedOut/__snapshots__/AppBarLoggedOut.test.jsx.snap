// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<AppBar
  iconElementLeft={
    <IconMenu
      anchorOrigin={
        Object {
          "horizontal": "left",
          "vertical": "top",
        }
      }
      animated={true}
      clickCloseDelay={200}
      iconButtonElement={
        <IconButton
          disableTouchRipple={false}
          disabled={false}
          iconStyle={Object {}}
          tooltipPosition="bottom-center"
          touch={false}
        >
          <NavigationMenu />
        </IconButton>
      }
      multiple={false}
      onClick={[Function]}
      onItemClick={[Function]}
      onKeyboardFocus={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onRequestChange={[Function]}
      open={null}
      targetOrigin={
        Object {
          "horizontal": "left",
          "vertical": "top",
        }
      }
      useLayerForClickAway={false}
    >
      <MenuItem
        anchorOrigin={
          Object {
            "horizontal": "right",
            "vertical": "top",
          }
        }
        checked={false}
        desktop={false}
        disabled={false}
        focusState="none"
        insetChildren={false}
        primaryText="About us"
        targetOrigin={
          Object {
            "horizontal": "left",
            "vertical": "top",
          }
        }
      />
      <MenuItem
        anchorOrigin={
          Object {
            "horizontal": "right",
            "vertical": "top",
          }
        }
        checked={false}
        desktop={false}
        disabled={false}
        focusState="none"
        insetChildren={false}
        primaryText="Create Account"
        targetOrigin={
          Object {
            "horizontal": "left",
            "vertical": "top",
          }
        }
      />
      <MenuItem
        anchorOrigin={
          Object {
            "horizontal": "right",
            "vertical": "top",
          }
        }
        checked={false}
        desktop={false}
        disabled={false}
        focusState="none"
        insetChildren={false}
        primaryText="Login"
        targetOrigin={
          Object {
            "horizontal": "left",
            "vertical": "top",
          }
        }
      />
    </IconMenu>
  }
  showMenuIconButton={true}
  title="Team Real Time"
  zDepth={1}
/>
`;
